VultisigApp/VultisigApp/Utils/Encryption.swift

```diff
@@ -7,6 +7,13 @@
 
 import Foundation
 import CommonCrypto
+import CryptoKit
+
+enum AESError: Error {
+    case keyGeneration
+    case encryptionFailed
+    case decryptionFailed
+}
 
 public extension String {
```

Comment:
May be it should be methods of some object? Instead of string extension. 

Like: 
```
struct Encryption {
let key: Data
func aesEncryptGCM(payload: String)
func aesEncrypt(payload: String)
}
```

Reply:
will leave it as it is for now , and maybe refactor it later